quickstart/frontend/vite.config.ts

```diff
@@ -1,28 +1,30 @@
-import {defineConfig} from 'vite'
+import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 import ViteYaml from '@modyfi/vite-plugin-yaml'
 
+const portFromEnv = process.env.APP_PROVIDER_UI_PORT || '3000'
+
 export default defineConfig({
     plugins: [
         react(),
         ViteYaml(),
     ],
     server: {
         host: true,
+        port: 3000,
```

Comment:
```suggestion
        port: ${portFromEnv},
```
can we avoid hardcoding ports? In my branch I cleaned up all the hardcoded ports please do not introduce new places that would need to be fixed

Reply:
Sure. Where do you want this to be defined? Needs to be different from `APP_PROVIDER_UI_PORT`.

Reply:
What is this port used for @iggy-da - is it specifically for hotload functionality, or does it have broader application?

Reply:
Specifically for hotload functionality. It was previously implicitly hard-coded 5173 (vite default) but we need it to be 3000 to the auth redirects to work, since they expect port 3000, which is also hard-coded in the auth config...